python_scripts/cross_validation_ex_01.py

```diff
@@ -65,11 +65,11 @@
 # As previously mentioned, the parameter `gamma` is one of the parameter
 # controlling under/over-fitting in support vector machine with an RBF kernel.
 #
-# Compute the validation curve
-# (using [`sklearn.model_selection.validation_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.validation_curve.html))
-# to evaluate the effect of the parameter `gamma`. You can vary its value
-# between `10e-3` and `10e2` by generating samples on a logarithmic scale.
-# Thus, you can use `np.logspace(-3, 2, num=30)`.
+# Evaluate the effect of the parameter `gamma` by using a
+# [`validation_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.validation_curve.html)).
```

Comment:
I think it's helpful to put the full module path in the question to make it easier to know where to import it from.

I know, the student could just look up the import statement from the previous notebook but since switching between notebooks is slow in Fun, let's be helpful here.

```suggestion
# [`sklearn.model_selection.validation_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.validation_curve.html)).
```